Update cancel button

frederoni opened the pull request:
The hitbox now also covers the entire space.

@bsudekum @ericrwolfe 

![screenshot 2017-09-07 17 14 13](https://user-images.githubusercontent.com/764476/30170656-0766253c-93f0-11e7-8692-ae6fa7fa31d2.png)
![screenshot 2017-09-07 17 14 25](https://user-images.githubusercontent.com/764476/30170658-08bf11b4-93f0-11e7-9fe7-dd5ca5414f19.png)

bsudekum commented:
Can any translations be removed?

bsudekum commented:
Do you think it makes sense to keep the tint in line with rest of the button tints? Looking specifically at the overview, feedback and mute buttons.

![image](https://user-images.githubusercontent.com/1058624/30182006-485ab466-93e3-11e7-98c0-884f81c9fbf2.png)

frederoni commented:
@bsudekum the tint color is usually don't applied to destructive or cancellation buttons.
I replaced the icon with a slightly more subtle one.


<img src="https://user-images.githubusercontent.com/764476/30281820-bf9d7bf2-9713-11e7-9a7c-8008e9dd8472.png" width=200><img src="https://user-images.githubusercontent.com/764476/30281821-c0945fb2-9713-11e7-887b-1d96bde06a03.png" width=200>

Also fixed the localization extraction script to include the storyboard and discard all "DO NOT TRANSLATE" strings.
/cc @1ec5
